src/content/en/2020/http2.md

```diff
@@ -69,48 +66,110 @@ Last year’s analysis of HTTP Archive data showed that HTTP/2 was used for over
-
-The drive to increase security and privacy on the web has seen requests over TLS increase by over 150% in the last 4 years. Today, over 86% of all requests on mobile and desktop are encrypted. Looking just at web site home pages, the numbers are still an impressive, 78.1% of desktop and 74.7% of mobile. This is important because HTTP/2 is only supported by browsers over TLS. The proportion served over HTTP/2, as shown in Figure 22.5, has also increased 10% from last year, from 55% to 65%.
+gQUIC is not seen in the home page data for two reasons. To measure a website over gQUIC, the HTTP Archive crawl would have to perform protocol negotiation via the [alternative services](#alternative-services) header and then use this endpoint to load the site over gQUIC. This was not supported this year, but expect it to be available in next year's Web Almanac. Also, gQUIC is predominantly used for third-party Google tools rather than serving home pages.
 
-<figure markdown>
-| Protocol | Desktop | Mobile |
-| -------- | ------- | ------ |
-| HTTP/1.1 | 36.05%  | 34.04% |
-| HTTP/2   | 63.95%  | 65.96% |
+{# TODO(authors, analysts): Where does the 150% stat come from? Can we link to it or show it as a figure? #}
+The drive to increase [security](./security) and [privacy](./privacy) on the web has seen requests over TLS increase by over 150% in the last 4 years. Today, over 86% of all requests on mobile and desktop are encrypted. Looking only at home pages, the numbers are still an impressive 78.1% of desktop and 74.7% of mobile. This is important because HTTP/2 is only supported by browsers over TLS. The proportion served over HTTP/2, as shown in Figure 22.5, has also increased by 10 percentage points from [last year](../2019/http2#fig-5), from 55% to 65%.
 
-<figcaption>{{ figure_link(caption="HTTP version usage for TLS home pages.", sheets_gid="900140630", sql_file="tls_adoption_by_http_version.sql") }}</figcaption>
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>Protocol</th>
+        <th>Desktop</th>
+        <th>Mobile</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>HTTP/1.1</td>
+        <td class="numeric">36.05%</td>
+        <td class="numeric">34.04%</td>
+      </tr>
+      <tr>
+        <td>HTTP/2</td>
+        <td class="numeric">63.95%</td>
+        <td class="numeric">65.96%</td>
+      </tr>
+    </tbody>
+  </table>
+  {# TODO(authors): Last year's version of this stat (linked above) used "HTTPS home pages" instead of "TLS home pages". Are they interchangeable? Is it better to be consistent or is TLS more correct? #}
```

Comment:
@dotjs

Reply:
This is one which @rmarx and I have argued with in the past 😁

I prefer HTTPS unless talking specifically about the TLS protocol features (ciphers, ALPN...etc.). I feel it's friendlier, more atune to what the reader sees in the URL bar and avoids the whole SSL/TLS noise and mouthful (SSL is now inaccurate but is more widely known than TLS to non-techies, SSL/TLS is too much of a mouthful).

Also what we use in HTTP Archive: https://httparchive.org/reports/state-of-the-web#pctHttps

Reply:
Yeah, I agree with @bazzadp on this one. While TLS is more "correct", HTTPS covers the load well enough here and is more approachable for the wide audience the Almanac is targeting. 

Though I still feel a book on HTTP/2 should use TLS instead, Barry ;)